### PR TITLE
Replace `project` with `package` in Learn pages

### LINKED ISSUE #4371: Confusion over Project vs Package
## Description
In the Ballerina getting started doc[1], it uses the word "project" everywhere. But everywhere else, I can see the word "package" has been used.  This confuses first-tome users and they infer these 2 as 2 different terms, but without finding a difference mentioned anywhere. (I myself was confused as the Ballerina hackathon asked to create a package, but the getting started doc doesn't mention "package" at all.) So better to stick to one term everywhere.

[1] https://ballerina.io/learn/get-started-with-ballerina/

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Replace `project` with `package` in Learn pages.
> Fixes #4371 

## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: learn/get-started/write-a-graphql-api-with-ballerina.md [hunk 1]
```diff
@@ -46,9 +46,9 @@ The `Query` type has two fields.
 The `Mutation` type has a single field.
 * The `add` field - This field will add a given entry to the data source.
 
-## Create a datasource for the project
+## Create a datasource for the package
```
Comment (sameerajayasoma) [inline learn/get-started/write-a-graphql-api-with-ballerina.md:49]: Let's keep project usage here. I don't think we should change it.

File: learn/get-started/write-a-graphql-api-with-ballerina.md [hunk 2]
```diff
@@ -46,9 +46,9 @@ The `Query` type has two fields.
...
 
-Before writing the GraphQL service, let's create a data source for the project. This will mimic a database that stores
+Before writing the GraphQL service, let's create a data source for the package. This will mimic a database that stores
 the data for the service. In this guide, you are going to use an in-memory table in Ballerina as the data source.
```
Comment (sameerajayasoma) [inline learn/get-started/write-a-graphql-api-with-ballerina.md:52]: Same comment as above

**Review by sameerajayasoma — CHANGES_REQUESTED** [review]

File: learn/get-started/write-a-graphql-api-with-ballerina.md (hunk 1 above)
Comment (praneesha) [reply learn/get-started/write-a-graphql-api-with-ballerina.md]: Updated in https://github.com/ballerina-platform/ballerina-dev-website/pull/4524/commits/848fe2d2779e36676e21b6d44eebf5c17f076c96.

File: learn/get-started/write-a-graphql-api-with-ballerina.md (hunk 2 above)
Comment (praneesha) [reply learn/get-started/write-a-graphql-api-with-ballerina.md:52]: Updated in https://github.com/ballerina-platform/ballerina-dev-website/pull/4524/commits/848fe2d2779e36676e21b6d44eebf5c17f076c96.

**Review by sameerajayasoma — APPROVED** [review]

**Pull request merged**